Core/include/Acts/EventData/SingleFreeTrackParameters.hpp

```diff
@@ -68,28 +68,25 @@ class SingleFreeTrackParameters {
   /// @brief Access all parameters
   ///
   /// @return Vector containing the store parameters
-  FreeVector parameters() const { return m_oParameters.getParameters(); }
+  ParametersVector parameters() const { return m_oParameters.getParameters(); }
 
   /// @brief Access to a single parameter
   ///
   /// @tparam kIndex Identifier of the parameter index which will be retrieved
   ///
   /// @return Value of the requested parameter
-  template <FreeParametersIndices kIndex,
-            std::enable_if_t<kIndex<eFreeParametersSize, int> = 0> ParValue_t
-                get() const {
+  template <FreeParametersIndices kIndex>
+  Scalar get() const {
     return m_oParameters.template getParameter<kIndex>();
   }
 
   /// @brief Access track parameter uncertainty
   ///
-  /// @tparam par Identifier of the parameter uncertainty index which will
-  /// be retrieved
+  /// @tparam kIndex Identifier of the parameter index which will be retrieved
```

Comment:
```suggestion
  /// @tparam kIndex Identifier of the uncertainty index which will be retrieved
```

Reply:
I'm not sure about the naming but the `parameter index` sounds a bit weird to me